bundles/io.github.linkedfactory.service/src/main/resources/OSGI-INF/io.github.linkedfactory.service-default.ttl

```diff
@@ -8,4 +8,4 @@
 <> <defaultModel> <http://linkedfactory.github.io/data/> .
 # <> <store> [ a <KvinLevelDb> ; <dirName> "linkedfactory-valuestore" ] .
 
- <store> [ a <KvinPartitioned> ; <archivePath> "LF-partition" ; <cycle> 10 ] .
\ No newline at end of file
+<> <store> [ a <KvinPartitioned> ; <archivePath> "LF-partition" ; <cycle> 10 ] .
```

Comment:
- We should align dirName and archivePath by deciding for one of the options.
- Furthermore, we should not use mixed upper- and lowercase in path names to better support Windows and Linux filesystems.
- Maybe "cycle" could also be renamed to "archiveInterval" or something like this.
- To be backwards compatible it would be good to use KvinLevelDb as default option. What do you think?

Reply:
#`To be backwards compatible it would be good to use KvinLevelDb as default option. What do you think?`
as i see in` KvinManager` KvinLevelDb  is already the default (if no store is configured). or do you mean somthing else?